tools/vsextension/ProjectWizard/source.extension.vsixmanifest

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="OpenEnclaveVisualStudioExtension-1" Version="0.7.28" Language="en-US" Publisher="Microsoft" />
+        <DisplayName>Open Enclave Wizard - Preview</DisplayName>
+        <Description xml:space="preserve">Support for developing and debugging Trusted Execution Environment enclaves that can run on both SGX and TrustZone, and using them from your own applications.</Description>
+        <MoreInfo>https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/visualstudio_dev.md</MoreInfo>
+        <License>LICENSE.txt</License>
+    </Metadata>
+    <Installation AllUsers="true">
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.0, 17.0)" />
```

Comment:
Does the `Pro` at the end restrict the edition?

Reply:
Tested the extension on a clean install of the Community edition and it installed fine.